catalog/spatial/registry/registry-publication-update-handler/src/main/java/org/codice/ddf/registry/publication/RegistryPublicationHandler.java

```diff
@@ -74,7 +74,7 @@ public void handleEvent(Event event) {
                 .contains(RegistryConstants.REGISTRY_TAG)) {
             return;
         }
-        executor.execute(() -> processUpdate(mcard));
+        executor.schedule(() -> processUpdate(mcard), 3, TimeUnit.SECONDS);
```

Comment:
I don't think so. The update needs to go out but a few seconds delay shouldn't make any to a remote system.